composer.json

```diff
@@ -11,10 +11,12 @@
             "email": "dcramer@gmail.com"
         }
     ],
+    "minimum-stability": "dev",
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.1",
         "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
+        "nokitakaze/testhttpserver": "^0.1.0",
```

Comment:
Unit tests test that something works as expected by mocking objects, not by calling a real server. Those are integration tests, and I can't see why we would need them here given that what a server replies is tested in the Sentry codebase itself and it's not our responsibility.